reference/docs-conceptual/learn/security-features.md

```diff
@@ -0,0 +1,95 @@
+---
+description: PowerShell has several features designed to improve the security of your scripting environment.
+ms.date: 09/23/2021
+title: PowerShell security features
+---
+# PowerShell security features
+
+PowerShell has several features designed to improve the security of your scripting environment.
+
+## Execution policy
+
+PowerShell's execution policy is a safety feature that controls the conditions under which
+PowerShell loads configuration files and runs scripts. This feature helps prevent the execution of
+malicious scripts. You can use a Group Policy setting to set execution policies for computers and
+users. Execution policies only apply to the Windows platform.
+
+For more information see [about_Execution_Policies][exe-policy].
+
+## Module and script block logging
+
+Module Logging allows you to enable logging for selected PowerShell modules. The setting is
+effective in all sessions on the computer. The pipeline execution events for the specified modules
+are recorded in the Windows PowerShell log in Event Viewer.
```

Comment:
```suggestion
Module Logging allows you to enable logging for selected PowerShell modules. This setting is
effective in all sessions on the computer. Pipeline execution events for the specified modules
are recorded in the Windows PowerShell log in Event Viewer.
```

MInor changes for easier reading and a question:
Is it only the `Windows PowerShell` log the correct location for all versions?